src/components/footer/Footer.css

```diff
@@ -0,0 +1,44 @@
+.footer{
+    width: 100%;
+    height: auto;
+    background-color: var(--dark-blue);
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+    align-items: center;
+    margin-top: 4rem;
+}
+
+.footer-container{
+    display: flex;
+    gap: 35rem;
+    padding-top: 4rem; 
+    flex-wrap: wrap;
+}
+
+.footer__item{
+   
+}
```

Comment:
came across this empty class. Remove it if not needed

Reply:
Yes, removed that class :)

Reply:
> The code looks clean, good job so far. Also, you'll implement router later right? Replace the anchor tags then.

Yes, I will React Router in future.